lib/query-builder.js

```diff
@@ -2,54 +2,70 @@
 
 var _ = require('lodash')
 var OpParser = require('./operator_parser')
-var StandardQuery = require('seneca-standard-query')
 
-function parseExtendedExpression (entp, sTypes, current_name, current_value, params, values) {
-  function parseComplexSelectOperator (sTypes, current_name, current_value, params, values) {
-    var result = {
-      processed: false
-    }
+module.exports = function (seneca, opts) {
+  var StandardQuery
+  seneca.ready(function () {
+    StandardQuery = seneca.export('standard-query/utils')
+  })
 
-    for (var op in current_value) {
-      var op_val = current_value[op]
-      if (!OpParser[op]) {
-        result.err = 'This operator is not yet implemented: ' + op
-        return result
+  function parseExtendedExpression (sTypes, currentName, currentValue, params, values) {
+    function parseComplexSelectOperator (sTypes, currentName, currentValue, params, values) {
+      var result = {
+        processed: false
       }
-      var err = OpParser[op](current_name, op_val, params, values, sTypes)
-      if (err) {
-        result.err = err
-        return result
+
+      for (var op in currentValue) {
+        var op_val = currentValue[op]
+        if (!OpParser[op]) {
+          result.err = 'This operator is not yet implemented: ' + op
+          return result
+        }
+        var err = OpParser[op](currentName, op_val, params, values, sTypes)
+        if (err) {
+          result.err = err
+          return result
+        }
+
+        result.processed = true
```

Comment:
Is this working ok? If the first operator is ok (processed=true) and the second one fails you return `processed:true`.

How about:
```
      var result = {}

      result.processed = _.every(currentValue, function (opVal, op) {
        if (!OpParser[op]) {
          result.err = 'This operator is not yet implemented: ' + op
          return false
        }

        var err = OpParser[op](currentName, opVal, params, values, sTypes)
        if (err) {
          result.err = err
          return false
        }

        return true
      })

      return result
```